cdap-standalone/src/main/resources/cdap-site.xml

```diff
@@ -268,4 +268,78 @@
     </description>
   </property>
 
+  <!-- Override default 0.0.0.0 to 127.0.0.1 (localhost) for all http services
+  to protect from xss vulnerabilities while running in sandbox -->
+  <property>
+    <name>dashboard.bind.address</name>
+    <value>127.0.0.1</value>
+    <description>
+      CDAP UI bind address
+    </description>
+  </property>
+
+  <property>
```

Comment:
For the services besides router and dashboard, isn't there already logic to do that here:
https://github.com/caskdata/cdap/blob/develop/cdap-standalone/src/main/java/co/cask/cdap/StandaloneMain.java#L444-L455

Reply:
Sweet!
Updated the PR and added one which was missing in standalone main.